src/app/series-import/series-import.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ModalService, TabService, ToastrService } from 'ngx-prx-styleguide';
+import { IMPORT_SERIES_VALIDATIONS } from '../shared/model/series.model';
+import { SeriesImportModel, ImportValidationState } from '../shared';
+
+import { SeriesComponent } from '../series/series.component'
+
+@Component({
+  providers: [],
+  selector: 'app-series-import',
+  templateUrl: './series-import.component.html',
+  styleUrls: ['./series-import.component.css']
+})
+export class SeriesImportComponent extends SeriesComponent implements OnInit {
```

Comment:
Good question!  Sounds like subclassing is not a common Angular practice?

I'm don't know how to use an Output or a Service. I'll take a look at that. Sounds like I could just pass that validation strategy in, sort of like a parameter to the component?

My thinking was to avoid having podcast import related logic in the SeriesComponent, while at the same time keeping the ability to leverage the SeriesComponent's built in facilities for handling Series form/model related functionality. For example, there was some talk about extending the import RSS form to include selecting an account for which that imported series would be attached.

Reply:
The other wrinkle here is the validate then save logic:
https://github.com/PRX/publish.prx.org/pull/538/files/4c227a9cce6fec8703d67b398d94ae048346a0f5#diff-dab4b34929e44150f90758ad805b00edR50

I looked at some conditional observable chaining--basically injecting the validation observable into a chain that it resolves in serial. 
There is an example here:
https://github.com/PRX/publish.prx.org/blob/master/src/app/shared/model/story.model.ts#L78
Looking at that and other examples, It seems like I'll have to set up some branching code (via the Output / Service) for that observable chain in the SeriesComponent's save method.